src/segments/svn.go

```diff
@@ -101,6 +102,13 @@ func (s *Svn) setSvnStatus() {
 		s.Branch = branch[2:]
 	}
 
+	// Get the repository name as the last path element of the repository root URL
+	repo := s.getSvnCommandOutput("info", "--show-item", "repos-root-url")
```

Comment:
If you make this a function called `Repo`, you can avoid this being called every time, it's only relevant when the user adds it to a template.

Reply:
Makes sense, thanks for the input!
Improved in commit 2716c490d016157ba4845a0375634026b3d55517.

Reply:
Makes sense, thanks for the input.
Resolved in commits 2716c490d016157ba4845a0375634026b3d55517 and 1b71249acc9e4abffc2d071dfa20755172cc30bb.